arches/app/media/js/views/components/search/map-filter.js

```diff
@@ -601,6 +601,16 @@ define([
                     maxZoom: maxZoom
                 });
             }
+        },
+        
+        toggleAriaExpanded: function(id) {
```

Comment:
@phudson-he I added a module in the js/utils directory called aria.js.  You should be able to import it at the top of this file like so `'utils/aria'` and then replace this function with this line and just use it like you have done in the templates.
`toggleAriaExpanded: ariaUtils.toggleAriaExpanded`

Reply:
Changes made in commit 98ce1b5